spec/components/shared/filters_component_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.describe Shared::FiltersComponent, type: :component do
+  let(:f) { VacancyAlgoliaSearchForm.new() }
```

Comment:
`f` should be an instance of `GOVUKDesignSystemFormBuilder::FormBuilder`

Reply:
ah ok thanks, give it ago

Reply:
Maybe something like this?
```ruby
GOVUKDesignSystemFormBuilder::FormBuilder.new(:search_form, VacancyAlgoliaSearchForm.new, rendered_component, {})
```

Reply:
This is what `f` needs to refer to https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-c-new

Reply:
i stubbed the formbuilder object instead as i couldnt get your suggestion to work?!